scripts/configure.sh

```diff
@@ -24,6 +24,17 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   CPUS=$(grep -c ^processor /proc/cpuinfo)
 elif [[ "$OSTYPE" == "darwin"* ]]; then
   CPUS=$(sysctl -n hw.ncpu)
+  # ensure development environment is set correctly for clang
```

Comment:
I do love the idea of simplifying down so that the workflow remains mostly just:

1. configure
2. build
3. run

(harkening back to the `Makefile` days of old). @davebryson does this seem like a reasonable simplification to you?

Reply:
Yes.  Looks good to me!